axelrod/strategies/shortmem.py

```diff
@@ -0,0 +1,36 @@
+from axelrod import Actions, Player
+from axelrod.actions import Action
+
+C, D = Actions.C, Actions.D
+
+class ShortMem(Player):
+    """A player who only ever defects."""
+
+    name = 'ShortMem'
+    classifier = {
+        'memory_depth': 10,
+        'stochastic': False,
+        'makes_use_of': set(),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    @staticmethod
+    def strategy(opponent: Player) -> Action:
+        
+        memoryDepth = self.classifier['memoryDepth']
+        cooperateRatio = array.count('C')/memoryDepth
```

Comment:
I though about that, but we need the number of cooperations and defections in the last 10 steps only, and to the extent that I understand it, it records the `C`s and `D`s for the whole play.